app/src/main/java/io/appium/uiautomator2/handler/NewSession.java

```diff
@@ -30,4 +37,19 @@ public AppiumResponse safeHandle(IHttpRequest request) {
         return new AppiumResponse(sessionID, WDStatus.SUCCESS, "Created Session");
     }
 
+
+    public  Map<String, Object> getCapabilities(IHttpRequest request) throws JSONException {
+        JSONObject caps = getPayload(request).getJSONObject("desiredCapabilities");
+        Map<String, Object> map = new HashMap<String, Object>();
+
```

Comment:
seems `entrySet() on JSONObject` available in `GSON(com.google.gson.JsonObject)` library but here we are using `org.json.JSONObject`